### PR TITLE
Dde light mode support

### PR DESCRIPTION
**Pull request by warunicorn19**
**Description**

This PR fixes #
Added a function to identify whether the user's system is using light mode or dark mode, based on that the color of the logo changes

Screenshot 2022-04-28 at 8 14 48 PM

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Will you also change the background color of the three rounded rectangles?

File: install/docker-extension/ui/src/components/ExtensionComponent/ExtensionComponent.js [hunk 1]
```diff
@@ -165,40 +172,40 @@ const ExtensionsComponent = () => {
           </ExtensionWrapper>
           <ExtensionWrapper>
             <div>
-              <CustomTypography sx={{ marginBottom: "1rem" }}>Deploy a Service Mesh</CustomTypography>
+              <Typography sx={{ marginBottom: "1rem", color:"white" }}>Deploy a Service Mesh</Typography>
```
Comment (leecalcote) [inline install/docker-extension/ui/src/components/ExtensionComponent/ExtensionComponent.js:175]: Does the Docker theme not have the light color defined already?

Comment (warunicorn19) [reply install/docker-extension/ui/src/components/ExtensionComponent/ExtensionComponent.js:175]: It does, i had to set it to white cause we are using a dark background. After switching to a lighter background for dark mode we can remove this.

Comment (leecalcote) [reply install/docker-extension/ui/src/components/ExtensionComponent/ExtensionComponent.js:175]: ok

**Pull request merged**